Controller/ContentManagement/CrudController.php

```diff
@@ -358,4 +374,47 @@ public function getContentTypeInfo(ContentType $contentType)
                 'contentType' => $contentType,
         ]);
     }
+
+    /**
+     * @Route("/api/user-profile", defaults={"_format": "json"}, methods={"GET"})
+     */
+    public function getUserProfile() : JsonResponse
```

Comment:
```suggestion
    public function getUserProfile(): JsonResponse
```

We should get this reported by phpCS!